raiden/ui/cli.py

```diff
@@ -1316,38 +1306,30 @@ def _run_smoketest():
             to_canonical_address(TEST_PARTNER_ADDRESS),
             TEST_DEPOSIT_AMOUNT,
         )
-
-        smoketest_config['contracts']['registry_address'] = to_checksum_address(
```

Comment:
the smoketest config was removed because it had old smart contract address in it, now we just use the freshly deployed addresses directly